01_Data/src/layers/sequelize/util.ts

```diff
@@ -40,36 +40,74 @@ import { MessageInfo } from './model/MessageInfo';
 import { Subscription } from './model/Subscription';
 import { Tariff } from './model/Tariff';
 import { IdTokenAdditionalInfo } from './model/Authorization/IdTokenAdditionalInfo';
-import { StatusNotification } from './model/Location/StatusNotification';
+import { StatusNotification } from './model/Location';
 
 export class DefaultSequelizeInstance {
   /**
    * Fields
    */
   private static instance: Sequelize | null = null;
+  private static logger: Logger<ILogObj>;
+  private static config: SystemConfig;
 
-  private constructor() {}
+  private constructor() {
+  }
 
   public static getInstance(config: SystemConfig, logger?: Logger<ILogObj>, sync: boolean = false): Sequelize {
```

Comment:
given that you've refactored the way you have, you can remove sync as an argument, which will remove the linter warning 🙌